tailcfg/tailcfg.go

```diff
@@ -650,9 +651,9 @@ type Service struct {
 	//        being a DNS proxy (when the node is an exit
 	//        node). For this service, the Port number is really
 	//        the version number of the service.
-	//     * "app-connector": the local app-connector service is
-	//        available. For this service, the Port number is
-	//        really the version number of the service.
+	//     * "app-connector": (deprecated) the local app-connector
+	//        service is available. For this service, the Port number
+	//        is really the version number of the service.
```

Comment:
btw, we used to have N version numbers in the client and consolidated to using CapVer for everything. (it used to be named MapRequest.Version or "mapver" but we renamed it mostly everywhere to "capver" to be more general)

Reply:
Yup, that makes sense. That was added mostly for consistency m.